api/src/main/java/uk/gov/dhsc/htbhf/claimant/ApiDocumentation.java

```diff
@@ -23,13 +29,19 @@
 @Slf4j
 public class ApiDocumentation {
 
-    @Value("${app.version:}") // use APP_VERSION env variable if available, otherwise give no version info
+    @Value("${app.version:1.0}") // use APP_VERSION env variable if available, otherwise default to 1.0
     private String appVersion;
 
+    @Autowired
+    private TypeResolver typeResolver;
+
     @Bean
     public Docket apiDocket() {
         return new Docket(DocumentationType.SWAGGER_2)
-                .host("N/A")
+                .alternateTypeRules(
+                        newRule(typeResolver.resolve(List.class, LocalDate.class),
+                                typeResolver.resolve(List.class, String.class)))
+                .host("localhost:8090")
```

Comment:
This should probably be dynamic and get port number from config?

Reply:
We thought about that and realised that whatever host/port number spring think's it's service from will almost certainly be hidden behinds a router somewhere - so this will only ever work on a local machine.